README.md

```diff
@@ -39,12 +32,45 @@ def application do
 end
 ```
 
+## If you'd like to mount VerkWeb on another Endpoint:
+
+
+```elixir
+defmodule MyApp.Endpoint do
+  use VerkWeb.Mount, path: "/verk"
+  ...
+end
+```
+
+That should be it! :)
```

Comment:
Oh Ok. I would like to keep it `server: true` as it's the easiest way to use `verk_web`. If the user wants to put it into their Phoenix Web app then the user can disable the server.